low-level/framework/messaging/egress_accumulated_msgs_processor.py

```diff
@@ -122,7 +128,12 @@ def run(self):
                             continue
                     if "sensor_response_type" in dict_msg["message"]:
                         logger.info(f"Publishing Accumulated Alert: {message}")
-                    self._producer.send([message])
+                    if isinstance(self._producer, MessageProducer):
+                        self._producer.send([message])
```

Comment:
create_MsgProducer_obj() here and then try send

Reply:
Why here??
So 1st time we are calling 'create_MsgProducer_obj()' in initialize() method, and 
here we are checking if self._producer instance is available, if it is not available then we are calling 'create_MsgProducer_obj()' function in else.

Reply:
As i understood in discussion, until delete in invoked, message from queue wont disappear and same message will be picked up as next element in same iteration also. in which case, alert order processing is not affected

Reply:
yes.